src/app/zap-templates/zcl/data-model/chip/microwave-oven-control-cluster.xml

```diff
@@ -18,7 +18,7 @@ limitations under the License.
   <domain name="CHIP"/>
 
 
-  <cluster apiMaturity="provisional">
```

Comment:
Actually, Q for @andreilitvin - does this need a provisional marker on the provisional feature?

Reply:
We have limited support for provisional across various items. We can do cluster and attribute at this point: https://github.com/project-chip/zap/blob/master/zcl-builtin/shared/schema/zcl.xsd#L482

Can we put some of the attributes provisional? It seems the SupportedWatts/SelectedWattIndex may be in that category.